base/bitset.jl

```diff
@@ -1,112 +1,237 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-struct BitSet <: AbstractSet{Int}
-    bits::BitVector
-    BitSet() = new(sizehint!(falses(0), 256))
+const Bits = Vector{UInt64}
+const Chk0 = zero(UInt64)
```

Comment:
I don't think you need this at all, you can just use `0` everywhere. The conversion will be implicit and it should be performed at compilation time.

Reply:
It was not really for performance, more for readability!

Reply:
Ok. I would still bikeshed the name though, to me it looks like a type when capitalized...

Reply:
Ok done, renamed to full caps, also for `NO_OFFSET`.